### ISSUE TITLE
Using the `increment` function results in metrics being created as gauges rather than counters 

### ISSUE THREAD

**Issue by eremzeit** [goodeggs/librato-node#44]
I would expect that `increment` should create counter metrics but it's creating gauges instead.

**Comment by eremzeit** [comment]
Whoops, I see this is a duplicate of https://github.com/goodeggs/librato-node/issues/36.

However, upon installing the newest version via ` npm install git+ssh://git@github.com:goodeggs/librato-node.git --save`, something seems misconfigured.  When I actually try to `var librato = require('librato-node');`, it gives me a:

```
Error: Cannot find module 'librato-node'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at /Users/erem/workspace/showgoers/server.js:336:15
    at Object.<anonymous> (/Users/erem/workspace/showgoers/server.js:378:3)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
```

This issue does not happen if I revert to version 2.

**Comment by eremzeit** [comment]
I've installed 4.0.1 and I'm still seeing this issue.

**Comment by bobzoller** [comment]
yikes, sorry about that.  appears I had a publishing error with 4.0.1.  I've deprecated that version and published 4.0.2.
